src/cpp/handler/handlerengine.cpp

```diff
@@ -1298,11 +1297,12 @@ class HandlerEngine::Private : public QObject
 		proxyStatsSock(0),
 		proxyStatsValve(0),
 		controlHttpServer(0),
-		stats(0),
-		report(0)
+		stats(0)
 	{
 		qRegisterMetaType<DetectRuleList>();
 
+		proxyControlClient = std::make_unique<ZrpcManager>(this);
```

Comment:
this already happens later and probably doesn't need to be here?

relatedly: watch out about that QObject parenting